### PR TITLE
Add systemd unit file example to agent setup docs

### PR DESCRIPTION
**Pull request by welteki**
Signed-off-by: Han Verstraete (OpenFaaS Ltd) <han@openfaas.com>

## Description

Add systemd unit file example to agent setup docs.

## Motivation and Context

Show how to run the agent as a systemd service.

## How Has This Been Tested?

Verified unit file and instructions.

## Types of changes

- [ ] Bug fix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)

## Checklist:

- [ ] My change requires a change to the documentation.
- [ ] I have updated the documentation accordingly.
- [x] I've read the [CONTRIBUTION](https://github.com/openfaas/faas/blob/master/CONTRIBUTING.md) guide
- [x] I have signed-off my commits with `git commit -s`

### CODE REVIEW

File: docs/add-agent.md [hunk 1]
```diff
@@ -183,6 +183,31 @@ Make sure you've read the [Actuated EULA](https://github.com/self-actuated/actua
...
 
+    An example of a systemd file you can use:
+
+    ```systemd
+    [Unit]
+    Description=Actuated Agent
+    [Service]
+    User=<user>
+    Type=simple
+    ExecStart=<user>/start.sh
```
Comment (alexellis) [inline docs/add-agent.md:194]: Let's specify a user and give a valid useradd command too.

File: docs/add-agent.md [hunk 2]
```diff
@@ -183,6 +183,31 @@ Make sure you've read the [Actuated EULA](https://github.com/self-actuated/actua
...
+    WantedBy=multi-user.target
+    ```
+
+    Check the `User` and `ExecStart` directives and make sure to replace `<user>` with the name of the user account for the agent. The usual place to save the unit file is: `/etc/systemd/system/actuated-agent.service`.
+
+    After saving your service file, you can start the service for the first time.
+
+    ```bash
+    sudo systemctl daemon-reload
+    sudo systemctl enable --now actuated-agent
```
Comment (alexellis) [inline docs/add-agent.md:208]: Shorter and easier to type if it's called `actuated` I think?

File: docs/add-agent.md [hunk 3]
```diff
@@ -183,6 +183,31 @@ Make sure you've read the [Actuated EULA](https://github.com/self-actuated/actua
...
+    Type=simple
+    ExecStart=<user>/start.sh
+    Restart=always
+    RestartSec=5s
+
+    [Install]
+    WantedBy=multi-user.target
+    ```
+
+    Check the `User` and `ExecStart` directives and make sure to replace `<user>` with the name of the user account for the agent. The usual place to save the unit file is: `/etc/systemd/system/actuated-agent.service`.
```
Comment (alexellis) [inline docs/add-agent.md:202]: Give them a command to copy so people don't have to think:

```
sudo cp actuated.service /etc/systemd/system/
```

File: docs/add-agent.md (hunk 2 above)
Comment (welteki) [reply docs/add-agent.md]: The service has been renamed to `actuated`.

File: docs/add-agent.md (hunk 1 above)
Comment (welteki) [reply docs/add-agent.md]: I updated the service file to run as root.

**Review by alexellis — APPROVED** [review]
Approved

**Pull request merged**